x-pack/plugins/fleet/server/routes/agent/index.ts

```diff
@@ -57,6 +57,7 @@ import { appContextService } from '../../services';
 import { postAgentUnenrollHandler, postBulkAgentsUnenrollHandler } from './unenroll_handler';
 import { FleetConfigType } from '../..';
 import { postAgentUpgradeHandler, postBulkAgentsUpgradeHandler } from './upgrade_handler';
+import { enforceSuperUser } from '../security';
```

Comment:
Along the same lines as my other comment, I'm not clear why some handlers use `enforceSuperUser` and some don't.

Can you add a comment with guidance for which should use it and which should not?